deploy/operator/common.sh

```diff
@@ -0,0 +1,35 @@
+__dir=${__dir:-"$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"}
+__root=${__root:-"$(realpath ${__dir}/../..)"}
+
+if [ -z "${DISKS:-}" ]; then
+    export DISKS=$(echo sd{b..f})
+fi
+
+export DISCONNECTED="${DISCONNECTED:-false}"
+export ASSISTED_DEPLOYMENT_METHOD="${ASSISTED_DEPLOYMENT_METHOD:-from_index_image}"
+export HIVE_DEPLOYMENT_METHOD="${HIVE_DEPLOYMENT_METHOD:-with_olm}"
+
+export OPENSHIFT_VERSIONS="${OPENSHIFT_VERSIONS:-$(cat ${__root}/data/default_ocp_versions.json)}"
+
+if [ -z "${ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE:-}" ]; then
+    export ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE=$(echo ${OPENSHIFT_VERSIONS} | jq -rc '[.[].release_image]|max')
+fi
+
+export LOCAL_REGISTRY="${LOCAL_REGISTRY_DNS_NAME}:${LOCAL_REGISTRY_PORT}"
+export RELEASE_IMAGE_REPOSITORY_TAG=${ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE#*/}
+export RELEASE_IMAGE_REPOSITORY=$(echo ${RELEASE_IMAGE_REPOSITORY_TAG} | cut -d':' -f1)
```

Comment:
where do we use this?

Reply:
It will be used in the next PR. I separated the disconnected changes from nit changes I made to the rest of the workflow.